addons/website/views/website_views.xml

```diff
@@ -104,6 +104,7 @@
                                 <field name="website_indexed"/>
                                 <field name="is_published"/>
                                 <field name="date_publish"/>
+                                <field name="website_meta_keywords"/>
```

Comment:
it is useful ? 
From previous Brainstorming, you need to see the content to set the right keyword, so frontend UI is enough.